pkgs/servers/tacacs+/default.nix

```diff
@@ -0,0 +1,21 @@
+{ stdenv, fetchurl, tcp_wrappers, flex, bison, perl }:
+
+stdenv.mkDerivation rec {
+  pname = "tacacs+";
+  version = "4.0.4.28";
+
+  src = fetchurl {
+    url = "ftp://ftp.shrubbery.net/pub/tac_plus/tacacs-F${version}.tar.gz";
+    sha256 = "17i18z3s58c8yy8jxp01q3hzz5nirs4cjxms18zzkli6ip4jszql";
+  };
+
+  buildInputs = [ tcp_wrappers flex bison perl ];
```

Comment:
`flex` and `bison` should be in `nativeBuildInputs`.

Reply:
Moved to `nativeBuildInputs`.